internal/service/arcregionswitch/generate.go

```diff
@@ -3,6 +3,5 @@
 
 //go:generate go run ../../generate/servicepackage/main.go
 //go:generate go run ../../generate/tags/main.go -ServiceTagsMap -KVTValues -ListTags -ListTagsInIDElem=Arn -ListTagsOutTagsElem=ResourceTags -UpdateTags -TagInIDElem=Arn -UntagInTagsElem=ResourceTagKeys
-// ONLY generate directives and package declaration! Do not add anything else to this file.
```

Comment:
This can be left in place - we keep it in all service generator files as a reminder for any new contributors.